addons/udes_stock_refactoring/models/stock_move.py

```diff
@@ -28,23 +28,22 @@ def _get_refactor_class(self, picking_type, stage):
         Return the relevant refactor class from the provided 
         picking type and stage, if applicable.
         """
-        try:
-            refactor_action = False
+        stage_post_action_dict = {
+            "confirm": picking_type.u_post_confirm_action,
+            "assign": picking_type.u_post_assign_action,
+            "validate": picking_type.u_post_validate_action,
+        }
 
-            if stage == "confirm":
-                refactor_action = picking_type.u_post_confirm_action
-            elif stage == "assign":
-                refactor_action = picking_type.u_post_assign_action
-            elif stage == "validate":
-                refactor_action = picking_type.u_post_validate_action
+        refactor_action = stage_post_action_dict.get(stage, None)
```

Comment:
Nit: None is default so not necessary - but feel free to leave in if you like the explicitness :)

Reply:
Was more me being too paranoid, will remove the `None`. :)

Reply:
where is stage? or am i being daft?

Reply:
ignore being daft

Reply:
Have now removed the unnecessary None default.